pkg/cmd/mom/output_resources_command.go

```diff
@@ -0,0 +1,68 @@
+package mom
+
+import (
+	"context"
+
+	"github.com/openshift/multi-operator-manager/pkg/library/libraryoutputresources"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+)
+
+func NewOutputResourcesCommand(streams genericiooptions.IOStreams) *cobra.Command {
+	return libraryoutputresources.NewOutputResourcesCommand(RunOutputResources, streams)
+}
+
+func RunOutputResources(ctx context.Context) (*libraryoutputresources.OutputResources, error) {
+	return &libraryoutputresources.OutputResources{
+		ConfigurationResources: libraryoutputresources.ResourceList{
```

Comment:
I don't understand the difference between Configuration, Management and UserWorkload Resources.